odd-platform-ui/src/redux/interfaces/state.ts

```diff
@@ -64,8 +64,7 @@ export interface TagsState {
   allIds: Tag['id'][];
   pageInfo?: CurrentPageInfo;
 }
-
-export interface LabelsState {
+export interface LabelsState extends EntityState<Label> {
```

Comment:
I think that we doesn't need `byId` and `allIds` fields with createEntityAdapter. You can remove them.